README.md

```diff
@@ -19,8 +19,8 @@ High level features include:
 - The schema should allow a data custodian to define the partner (organization or person) that each rule pertains to. For example, a certain rule may be applicable only to the Public Health Agency of Canada (PHAC) while another rule may be applicable to not only the PHAC but also to Ottawa Public Health.
 - The schema should allow data custodians to define rules that applies to rows or to columns. For example, a rule can be made to exclude all the rows from the `Sample` table or to exclude the `type` column from the `Sample` table.
 - Rules can be made within the context of an entire table, to a column that may be present in more than one table or to a column specific to a table.
-- The rules may only be exclusive. For example, rules can be defined to remove rows but not to include them.
-- Rules can be combined to form more powerful conditions using logical operators. For example, exclude all rows with `contactEmail` equal to "john.doe@email.com" AND `contactName` equal to "John Doe". Current supported logical operators are `AND`, `OR` and `NOT`.
+- The rules may only be exclusive. For example, rules can be defined to remove rows but not to include them. 
+- Rules can be combined to form more powerful conditions using logical operators. For example, exclude all rows with `contactEmail` equal to "john.doe@email.com" AND `contactName` equal to "John Doe". Current supported logical operators are `AND` and `OR`.
```

Comment:
I don't know why this was changed to exclude the `NOT` operator, but given that we're shifting approaches to the 'whitelist'/allowlist approach, it makes sense to drop it.